src/components/CircleButton/CircleButton.js

```diff
@@ -67,7 +67,7 @@ function getLabelColor(theme: Theme, isDisabled: boolean) {
   return colors.control;
 }
 
-const CircleButtonIconWrapperColors = ['#ffffff', '#f2f4f9'];
+const CircleButtonIconWrapperColors = ['#ffffff', '#f2f4f9']; // eslint-disable-line i18next/no-literal-string
```

Comment:
The plugin options accept regexes, so maybe all color codes could be ignored - I think the format is specific enough that it wouldn't match any UI text by accident.

Reply:
Good call! Added. :)